packages/serverless/src/apollo/app/user.ts

```diff
@@ -108,6 +114,11 @@ export const USER_FIND_REQUEST = Joi.object<TUserFindRequest>({
   pagination,
 });
 
+export const USER_SEARCH_REQUEST = Joi.object<TUserSearchRequest>({
+  query: Joi.object(),
```

Comment:
As we pass userName and email as string, can we validate these strings more specific. For example, Joi.string().min(1).max(255)

Reply:
Resolved